src/main/host/process.rs

```diff
@@ -1074,6 +1152,28 @@ impl Process {
                 .unwrap();
         }
 
+        // Initialize emulate rlimits to their native values.
+        // TODO: For determinism, we may want to use fixed limits for some or all of these.
+        // Some applications like Tor will change behavior depending on these limits.
+        // Ultimately they'd need to be compatible with the native limits though.
+        let mut rlimits: [linux_api::resource::rlimit64;
+            linux_api::resource::RLIM_NLIMITS as usize] =
+            [shadow_pod::zeroed(); linux_api::resource::RLIM_NLIMITS as usize];
+        for r in 0..linux_api::resource::RLIM_NLIMITS {
+            let r = linux_api::resource::Resource::try_from(r).unwrap();
```

Comment:
This seems a little fragile if linux increases `RLIM_NLIMITS` without adding a constant for each new value (or if there is some value that isn't exposed on x86-64). But it works currently, and if linux ever does cause this to break, it will fail loudly since it's in a common code path, so I think it's okay.